packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -109,6 +112,15 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     emit TobinTaxSet(value);
   }
 
+  /**
+   * @notice Sets the tobin tax.
+   * @param value The tobin tax.
+   */
+  function setTobinTax(uint256 value) public onlyOwner {
```

Comment:
Copy pasted this in again by accident, removed this duplicate block in next commit.